src/cavendish_particle_tracks/_main_widget.py

```diff
@@ -186,6 +187,17 @@ def hideEvent(self, event):
         self.viewer.window._qt_viewer.viewerButtons.show()
         super().hideEvent(event)
 
+    def _get_shuffling_seed(self, fallback=1):
```

Comment:
Sorry, I should have added type hints and a docstring...
```suggestion
    def _get_shuffling_seed(self, fallback: int=1) -> int:
        """Get the shuffling seed from the environment variable.
        
        This is useful, for example, for each lab computer being seeded differently.
        """
```